src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -66,28 +65,36 @@ export const Modal = (props: Props) => {
         return t('purchase.voucher', { seller: sellerName });
       case ModalPaymentTypes.modalPages.light_up_chinatown:
         return t('purchase.donation_to', { seller: sellerName });
+      case ModalPaymentTypes.modalPages.mega_gam:
+        return t(
+          'purchase.mega_gam',
+          sellerName ? { seller: sellerName } : { seller: 'Seller is blank' }
+        ); // todo: confirm which copy to use & who the seller is
       default:
         return t('purchase.donation', { seller: sellerName });
     }
   };
 
+  const getSubheaderText = (purchaseType) => {
+    switch (purchaseType) {
+      case ModalPaymentTypes.modalPages.donation:
+        return t('paymentProcessing.amount.header');
+      case ModalPaymentTypes.modalPages.gift_card:
+        return t('paymentProcessing.amount.header');
+      case ModalPaymentTypes.modalPages.light_up_chinatown:
+        return t('paymentProcessing.amount.light_up_chinatown');
+      case ModalPaymentTypes.modalPages.mega_gam:
+        return t('paymentProcessing.amount.mega_gam');
+      default:
+        return t('paymentProcessing.amount.header');
+    }
+  };
+
   return (
     <ContentContainer id="donation-form" data-testid="modal-amount">
       <Header>{getHeaderText(modalView, props.sellerName)}</Header>
-
-      {props.sellerId === 'send-chinatown-love' && (
```

Comment:
why are we deleting this `props.sellerId === 'send-chinatown-love' ` code?

Reply:
` {props.sellerId === 'send-chinatown-love' && (
        <p>{t('donationPool.description2')}</p>
      )}`

So `donationPool.description2` is equal to `All donations will be distributed evenly to our merchants.",` but this is not used anywhere in the checkout modal. In addition, it isn't evaluating to a value that's being displayed on the page (unless I misunderstood this line!)